### PR TITLE
Update CHANGELOG.md and README.md for release 0.3.0

### LINKED ISSUE #111: Commonalities release 0.3.0
**Problem description**
Release 0.2.0 has been published. New topics are being discussed: security scheme, linting rules, test definitions and other changes and clarifications in API Design Guidelines.

**Expected action**
Agree with TSC on Commonalities release schedule.

Define PRs and Issues that should be closed before the next release.

**Additional context**
This release can be considered as a base for common release of CAMARA subprojects.

### PR DESCRIPTION
**Pull request by rartych**
Changes for rel. 0.3.0

#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:
CHANGELOG.md and README.md for release 0.3.0

#### Which issue(s) this PR fixes:

Fixes #111

#### Changelog input

```
CHANGELOG.md and README.md for release 0.3.0

```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,47 @@
...
+    -  [Linting rules implementation files](https://github.com/camaraproject/Commonalities/tree/release-0.3.0/artifacts/linting_rules)
+
+### Added
+* API Testing Guidelines created by @shilpa-padgaonkar in https://github.com/camaraproject/Commonalities/pull/117
+* API Design Guidelines updated with scopes naming guidelines by @jlurien in https://github.com/camaraproject/Commonalities/pull/57
+* API Linting Rules - initial linting ruleset description by @rartych in https://github.com/camaraproject/Commonalities/pull/74
+* API linting implementation and guidelines by @ravindrapalaskar17 in https://github.com/camaraproject/Commonalities/pull/110
+  
+### Changed
+* CAMARA_common.json renamed to CAMARA_common.yaml and updated by @shilpa-padgaonkar in https://github.com/camaraproject/Commonalities/pull/107
```
Comment (shilpa-padgaonkar) [inline CHANGELOG.md:32]:
```suggestion
* CAMARA_common.yaml includes the following changes:

- CAMARA_common.json was changed to CAMARA_common.yaml to be consistent with all Camara API specs.
- Includes guidance for info object
- Adds the aligned device schema 
updated by @shilpa-padgaonkar in https://github.com/camaraproject/Commonalities/pull/107

```

**Review by bigludo7 — APPROVED** [review]
LGFM

**Review by shilpa-padgaonkar — APPROVED** [review]
/LGTM

**Comment by rartych** [comment]
@patrice-conil @RubenBG7 Could you kindly approve this last step for the release 0.3.0?

**Pull request merged**